### PR TITLE
move social_image cache to Redis

### PR DESCRIPTION
**Pull request by jeffin143**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
move social_image cache  to RedisRailsCache
## Related Tickets & Documents
handle #4670 

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/view_objects/articles/social_image.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def url
     def legacy_article_social_image
       cache_key = "article-social-img-#{article}-#{article.updated_at}-#{article.comments_count}"
```
Comment (mstruve) [inline app/view_objects/articles/social_image.rb]: Same as the other PR, can we reformat updated_at here to use `.rfc3339`? Please and THANK YOU!!!!

Comment (jeffin143) [reply app/view_objects/articles/social_image.rb]: Done :rocket:

File: app/view_objects/articles/social_image.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def url
     attr_reader :article
 
     def legacy_article_social_image
-      cache_key = "article-social-img-#{article}-#{article.updated_at}-#{article.comments_count}"
+      cache_key = "article-social-img-#{article}-#{article.updated_at&.rfc3339}-#{article.comments_count}"
```
Comment (rhymes) [inline app/view_objects/articles/social_image.rb]: I don't think we need the `&` for `article.updated_at`

Comment (jeffin143) [reply app/view_objects/articles/social_image.rb]: Resolved

**Pull request merged**